### PR TITLE
Reviewapps/codt2022

### PR DESCRIPTION
**Pull request by ShotaKitazawa**
# Description

None Issues

* Add reviewapp manifests for CODT2022
* bump reviewapp-operator to v0.0.10

## Type of change

- [ ] Document update or simple typo fix
- [x] Maintenance/update components
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] I have checked backward/forward compatibility that may cause regarding this change.

### CODE REVIEW

**Comment by ShotaKitazawa** [comment]
`manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/*.yaml` and `manifests/reviewapps/dreamkast_codt2022/.mt-stable/*.yaml` are almost copied from `manifests/app/dreamkast/overlays/development/template-dk/`

File: manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/deployment-dreamkast.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: dreamkast
+        tier: dreamkast
+    spec:
+      initContainers:
+      - name: initdb
+        image: 607167088920.dkr.ecr.ap-northeast-1.amazonaws.com/dreamkast-ecs:main
```
Comment (inductor) [inline manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/deployment-dreamkast.yaml:26]: ここ、変えなくて大丈夫だっけ

Comment (ShotaKitazawa) [reply manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/deployment-dreamkast.yaml]: `s/607167088920/632965872029/g` に修正しました

File: manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: dreamkast
+  labels:
+    app: dreamkast
+spec:
+  virtualhost:
+    fqdn: dreamkast-{{.Variables.AppRepositoryAlias}}-{{.AppRepo.PrNumber}}.dev.cloudnativedays.jp
```
Comment (inductor) [inline manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/ingress.yaml:9]: このドメインは意図してる？

Comment (ShotaKitazawa) [reply manifests/reviewapps/dreamkast-ui_codt2022/.mt-stable/ingress.yaml]: fqdn とその下の secret name も合わせて修正しました。

File: manifests/reviewapps/dreamkast_codt2022/.mt-stable/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      - name: dreamkast
+        port: 80
+    - conditions:
+      - prefix: /cable
+      enableWebsockets: true
+      services:
+      - name: dreamkast
+        port: 80
+    - conditions: #TODO: Need to resolve this hardcoded routing
+      - prefix: /o11y2022/ui
```
Comment (ShotaKitazawa) [inline manifests/reviewapps/dreamkast_codt2022/.mt-stable/ingress.yaml:25]: this description is referred to [cloudopsdays/dreamkast-infra_codt2022](https://github.com/cloudopsdays/dreamkast-infra_codt2022/blob/61f5a453358031592e6112ee83939c7b10a99955/manifests/app/dreamkast/overlays/production/main/ingress.yaml#L26)

**Pull request merged**